### DISCUSSION TITLE
ccf-validation-tools/Anatomical_Systems/comments/

### DISCUSSION THREAD

**Discussion by giscus** [hubmapconsortium/ccf-validation-tools#262] [General]
# ccf-validation-tools/Anatomical_Systems/comments/

https://hubmapconsortium.github.io/ccf-validation-tools/Anatomical_Systems/comments/

**Comment by emquardokus** [comment]
anatomical-systems-DRAFT-v1.0
https://docs.google.com/spreadsheets/d/1oB3m_IOMEcDsdQoDicXd9KXXz7oytIPIPiu9lq88quU/edit#gid=2028977062
**Different labels**
fixed circulatory system UBERON:0001009

**Blank ontology ID**
- [ ] In row 18, no term id was found for the name/label liver.
Added labels for liver UBERON:0002107
- [ ] IIn row 36, no term id was found for the name/label upper respiratory system.
Change to upper respiratory tract	UBERON:0001557
- [ ] IIn row 37, no term id was found for the name/label lower respiratory system.
Change to lower respiratory tract	UBERON:0001558

**Comment by emquardokus** [comment]
**anatomical-systems-v1.1_DRAFT**
https://docs.google.com/spreadsheets/d/13qyyCHS4lxgIfVtQJ_aVxB-aWch7oJG-psrZn4ZoFEM/edit#gid=2028977062

**Different labels**
Rows 49-55 Changed skeletal muscle organ to skeletal muscle organ, vertebrate

**Comment by emquardokus** [comment]
**anatomical-systems-v1.1_DRAFT**
https://docs.google.com/spreadsheets/d/13qyyCHS4lxgIfVtQJ_aVxB-aWch7oJG-psrZn4ZoFEM/edit?gid=2028977062#gid=2028977062

**Different labels**
1. FIXED kidney	UBERON:0002113;  The term UBERON:0011143 has a different name/label in the source ontology in the following 2 rows 44, 44. The name/label in the ASCT+B table is kidney and the one in the ontology is upper urinary tract. For reference, the given name/label by SMEs is kidney. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.
